app/models/reports/region_item.rb

```diff
@@ -0,0 +1,16 @@
+module Reports
+  class RegionItem
```

Comment:
Instead of a new `RegionItem`, can we just use `Item` with `@period = Time.current`? I'm assuming this is because we want to do all time things such as `cumulative_registrations`.

Reply:
Correct, for some of the Regions we want to grab _all_ periods of data back to the start of that Region's history.  For those, it doesn't make sense to have period inside the item.

Reply:
If the `Reports::Item` has a `@period= Time.current` we _could_ ask it to store `cumulative_registrations` as of today. I think we can optimise for one less interface if it's not too much of a stretch.

Reply:
> Could you try and see if it's possible to do this neatly without `RegionAndPeriodFetcher` and doing plain hash lookups?

Yup, I removed the fetcher and am doing a normal hash lookup now.